src/backend/renderer.js

```diff
@@ -1308,7 +1401,14 @@ export function attach(
             timestamp: interaction.timestamp - profilingStartTime,
           })
         ),
+        updaters:
```

Comment:
Note to self: We should probably respect filtering preferences here. Otherwise we may end up showing "an unmounted component" when it was really just a filtered-out component.